src/modules/data.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-restricted-globals */
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] Same as here too

Reply:
I really appreciate +1 your review. Thank You pray
Already fixed.